pull_request_template.md

```diff
@@ -7,25 +7,19 @@
 - [ ] Refactor
 - [ ] Other,describe:
 
-### Describe Key Changes
+### Describe Key Changes, Linked Tickets, etc
 
-### Related Tickets & Documents
-<!---  Related Jira ticket number and link -->
-<!---  Links to related documents -->
 
-### Request Reviewer
-<!--- Please select a team member to review before merging PR -->
-
-## Checklist For Opening a PR
+### Checklist For Opening a PR
 
 - [ ] Performed self-review of code
-- [ ] **Evaluated impacts on NWAC Avy App**
-  - [ ] Contacted NWAC dev team if applicable
+- [ ] Downstream consumers
```

Comment:
treat our widgets the same as avy app. even though we have control of api and widgets, this will put us in their shoes